### PR TITLE
Tag this repo too

### PR DESCRIPTION
**Pull request by maurolepore**

### CODE REVIEW

File: pacta/build_with_tag.sh [hunk 1]
```diff
@@ -69,7 +79,6 @@ do
     rm -rf "$repo"
 done
 
-parent="$(dirname $(which $0))"
 image_tar_gz="${parent}/2dii_pacta.tar.gz"
```
Comment (cjyetman) [inline pacta/build_with_tag.sh:82]: think you changed this syntax `${parent}`, to this syntax `$parent` everywhere else?

File: pacta/build_with_tag.sh [hunk 2]
```diff
@@ -38,17 +38,27 @@ then
     exit 2
 fi
 
+# Clone
 for repo in $repos
 do
-    # Clone
     remote="${url}${repo}.git"
```
Comment (cjyetman) [inline pacta/build_with_tag.sh:44]: oh, also here

**Comment by cjyetman** [comment]
and you'll fix this line in a different PR?
https://github.com/2DegreesInvesting/2diidockerrunner/blob/2a1d6ec005d99265ff306c770916fa368e29e256/pacta/build_with_tag.sh#L64

**Review by cjyetman — APPROVED** [review]
lgtm

File: pacta/build_with_tag.sh (hunk 1 above)
Comment (maurolepore) [reply pacta/build_with_tag.sh:82]: Yeah, here I was afraid of the "/". Should it be fine?

File: pacta/build_with_tag.sh (hunk 2 above)
Comment (maurolepore) [reply pacta/build_with_tag.sh:44]: I'm undecided. With my little experience `"$this $looks $safe"` but `"$this$looks$weird"` so I prefer `"${this}${syntax}"`.
Any suggestion?

**Comment by maurolepore** [comment]
Thanks! 
Yes, the fix to the dockerfile problem is in #26

File: pacta/build_with_tag.sh (hunk 1 above)
Comment (cjyetman) [reply pacta/build_with_tag.sh:82]: hmmmmm... good point... not sure

I guess this is also one of those places you should quote, quote, quote just to be safe, i.e. `""${parent}"/2dii_pacta.tar.gz"`

but don't take my word for it, experimenting/testing is the only way to know for sure I guess

File: pacta/build_with_tag.sh (hunk 2 above)
Comment (cjyetman) [reply pacta/build_with_tag.sh:44]: yeah, that makes sense to me.... if there's no whitespace around it, maybe safer to use `{}`

can totally see how `$foobar` is not the same as `${foo}bar`

**Pull request merged**